### PR TITLE
fix OCPQE-11756

### PR DESCRIPTION
**Pull request by yapei**
Before this change, I can see errors in local container
```
      unknown error: Chrome failed to start: exited abnormally.
        (unknown error: DevToolsActivePort file doesn't exist)
        (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.) (Selenium::WebDriver::Error::UnknownError)
      #0 0x56406407b693 <unknown>
      #1 0x564063e74b0a <unknown>
```

### CODE REVIEW

**Comment by yapei** [comment]
`element_scroll_behavior` options seems also need to be fixed, but I would like to fix it later since little case will be blocked

**Comment by yapei** [comment]
@jhou1  @chengzhang1016  @liangxia  @yanpzhan

**Comment by yapei** [comment]
In progress of verification

**Comment by yapei** [comment]
normal cluster pass log https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/619996/console

File: lib/webauto/web4cucumber.rb [hunk 1]
```diff
@@ -147,9 +147,7 @@ def browser
         if Integer === @scroll_strategy
           chrome_caps[:element_scroll_behavior] = @scroll_strategy
         end
-        if self.class.container?
```
Comment (liangxia) [inline lib/webauto/web4cucumber.rb:150]: Would you mind to try locally directly ( not in container ) ?

**Comment by yapei** [comment]
container [pass log](http://pastebin.test.redhat.com/1074837)
local [pass log](http://pastebin.test.redhat.com/1074834)

File: lib/webauto/web4cucumber.rb (hunk 1 above)
Comment (yapei) [reply lib/webauto/web4cucumber.rb:150]: @liangxia reverted these changes, PTAL

**Pull request merged**